ISISNeutronMuon/MDANSE
# Save input parameters in output files

**Description of work**
Every output file now contains the name of the job that created it, the version of MDANSE and a complete list of input parameters.

closes #283 

**Fixes**
OutputData now takes the IJob class instance as the last input.
HDFFormat and TextFormat both create an additional data group called "metadata".
The input parameters are written into the "metadata".
Plotter window puts the "metadata" into the tooltip of the main data item.
Trajectory tab puts the "metadata" into the trajectory info.
Converter class now adds the unit information to the output arrays.
LocalSession has been modified to store the basic information about the file paths of converters.
TrajectoryTab now shows a preview of the trajectory's time axis.
TrajectoryTab and Plotter can load multiple files.

**To test**
All tests must pass.
Also, please run a trajectory converter, then load the trajectory and check if the input parameters of the converter are shown in the trajectory info.
Next, run an analysis and load the results into the plotter. The input parameters should be shown in the tooltip on mouse hover.
